geode-core/src/main/java/org/apache/geode/distributed/internal/ClusterDistributionManager.java

```diff
@@ -1387,7 +1387,7 @@ void waitForViewInstallation(long id) throws InterruptedException {
       return;
     }
     synchronized (membershipViewIdGuard) {
-      while (membershipViewIdAcknowledged < id && !stopper.isCancelInProgress()) {
+      while (!stopper.isCancelInProgress()) {
```

Comment:
I think that this LGTM alert is incorrect, since it fails to take into account the value of `membershipViewIdAcknowledged` being modified by another thread, which seems to be what this method expects to happen. This false positive alert has been suppressed in this PR: https://github.com/apache/geode/pull/5473 to prevent it from showing up in future.